chromeBinary.dot

```diff
@@ -0,0 +1,9 @@
+digraph versiontest {
+
+  input [data="/usr/local/google/home/soonm/tree-builder-builder/example.ejs"]
+  chromeBinary [version="323860", chromium="/usr/local/google/home/soonm/chromium/src", platform="linux"]
+  telemetryPerf [chromium="/usr/local/google/home/soonm/chromium/src", browser="system", perfBrowser="system"]
```

Comment:
I don't think it's appropriate to check in absolute paths like this since it won't work on other computers. If we can't specify relative paths we should have placeholders instead like <path to tree-builder-builder>.